src/proto/CMakeLists.txt

```diff
@@ -1,6 +1,19 @@
 # Protobuf
-find_package(Protobuf)
+find_package(Protobuf REQUIRED)
+
 add_library(mixxx-proto OBJECT)
+
+include(IsStaticLibrary)
+if(TARGET protobuf::libprotobuf-lite)
+    target_link_libraries(mixxx-proto PRIVATE protobuf::libprotobuf-lite)
+    is_static_library(Protobuf_USE_STATIC_LIBS protobuf::libprotobuf-lite)
```

Comment:
I'm doubt this is even needed. Protobuf_USE_STATIC_LIBS is a parameter for the Protobuf find module (https://cmake.org/cmake/help/latest/module/FindProtobuf.html) so I don't think it does anything after the `find_package(Protobuf REQURIED)`.

Reply:
Oh yes. So lets remove it entirely. Do we have still a use case for setting it from Mixxx? If so, we can reintroduce it.